### ISSUE TITLE
Can't insert text into <input>s

### ISSUE THREAD

**Issue by lukasepple** [grindhold/rainbow-lollipop#28]
```
** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_application_get_state: assertion 'self != NULL' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_application_get_state: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed
/usr/share/rainbow-lollipop/restore_session.png

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_get_splits: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_x: assertion 'CLUTTER_IS_ACTOR (self)' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_width: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

** (rainbow-lollipop:7299): CRITICAL **: guint webkit_uri_response_get_status_code(WebKitURIResponse*): assertion 'WEBKIT_IS_URI_RESPONSE(response)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_get_splits: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_x: assertion 'CLUTTER_IS_ACTOR (self)' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_width: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:7299): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

** (rainbow-lollipop:7299): CRITICAL **: guint webkit_uri_response_get_status_code(WebKitURIResponse*): assertion 'WEBKIT_IS_URI_RESPONSE(response)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_index_of_child: assertion 'self != NULL' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_get_splits_until: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_y: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_index_of_child: assertion 'self != NULL' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_get_splits_until: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_y: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_index_of_child: assertion 'self != NULL' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_get_splits_until: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_y: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_index_of_child: assertion 'self != NULL' failed

** (rainbow-lollipop:7299): CRITICAL **: rainbow_lollipop_node_get_splits_until: assertion 'self != NULL' failed

(rainbow-lollipop:7299): Clutter-CRITICAL **: clutter_actor_get_y: assertion 'CLUTTER_IS_ACTOR (self)' failed
```

Seems to affect all websites, tested on wikipedia and github

**Comment by grindhold** [comment]
fixed as of 12930ae

**Comment by lukasepple** [comment]
awsm. will try it when back home.
